subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/ToBeFixedForInstantExecution.java

```diff
@@ -34,6 +34,8 @@
 
     Skip value() default Skip.DO_NOT_SKIP;
 
+    String[] bottomSpecs() default {};
```

Comment:
⭕️ At first the name got me confused but I see now that it's a Spock term so it's ok.